package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "test",
-    "start": "nodemon src/server/index.js"
+    "start": "nodemon src/server/index.js",
+    "build": "webpack -w"
```

Comment:
1. Definitely!
2. Once we determine whether to use dotenv and/or cross-env (see [issue #2](https://github.com/CSX-Collective/RallyPoint/issues/2)), I think we should definitely modify NODE_ENV inline in our scripts